src/cli/init.rs

```diff
@@ -67,6 +73,9 @@ pub enum InitKind {
 
     /// An empty model, suitable for a library or plugin.
```

Comment:
Do we want to update this comment? Model isn't really suited for plugins, and we have one that truly is now.